create_mode.go

```diff
@@ -0,0 +1,52 @@
+package zk
+
+import "fmt"
+
+const (
```

Comment:
Should we add a todo to change the API to give this type a name at some point? Then it'd be clear it's an enum and not the full int32 range

```
type CreateMode int32 // or CreateModeFlag or Flag or ModeFlag...
```

Reply:
left a todo for V2, i'm trying to limit the breaking changes. we could technically do an internal enum, but i think leaving this in the parse function to validate is ok for now.